docs/administrative-requirements.md

```diff
@@ -106,25 +101,50 @@ Resources:
-* [ESECC Self-service Portal (must be on VA network to view)](https://esecc.va.gov/CGWeb/Main.aspx) for initiating new requests or change requests
+The VA Systems Inventory (VASI) is intended to be a registry of all applications in use at VA. 
+
+The Enterprise Event Bus VASI ID is 3325, and [this is our entry (must be on VA network to view)](https://vaww.vear.ea.oit.va.gov/system_and_application_domain_defs_system_381405.htm) in the VASI registry. You may find your system’s VASI ID by visiting the [VA System Inventory (must be on VA network to view)](https://vaww.vear.ea.oit.va.gov/).
+
+**For non-LHDI clients**:
+
+* If a prospective client system does not have a VASI ID, it will need to submit a LEAF intake request to get one. 
+
+**For new LHDI clients**:
+
+* For new teams on Lighthouse Delivery Infrastructure (LHDI), please consult your LHDI Enablement Liaison to start the LEAF intake request and LHDI-specific onboarding. 
+
+### eMASS
+
+Enterprise Mission Assurance Support Service (eMASS) is VA’s Governance, Risk and Compliance (GRC) tool. All systems will be assessed in eMASS by the Risk Review team for an authorization recommendation to be submitted to the Authorizing Official (AO) for final ATO consideration.
+
+All VA systems are given an entry with a unique number in the eMASS system. The Enterprise Event Bus eMASS ID is 2350. 
+
+Getting an eMASS ID is a prerequisite for System Categorization. 
+
+**For non-LHDI clients**:
+
+* If a prospective client system does not have an eMASS ID, it will need to submit a LEAF intake (formerly GRC intake) request: [Unified System Registry Intake (must be on VA network to view)](https://leaf.va.gov/NATIONAL/103/cybersecurity_request_portal/) to get one.
+
+**For new LHDI clients**:
+
+* For new teams on Lighthouse Delivery Infrastructure (LHDI), please consult your LHDI Enablement Liaison to start the LEAF intake request process and LHDI-specific onboarding. 
 
 ## WASA Testing
 
 Web Application Security Assessment (WASA) scanning may be needed as part of the ATO process when a client system builds a web application for consuming events from the Event Bus. Please note that a 30 day notice is required for WASA testing.
 
-The VA SAVD WASA Coordination team can be contacted via email at [VASAVDWASACoordination@va.gov](mailto:VASAVDWASACoordination@va.gov). A request for WASA testing can be submitted on the [Security Assessment Portal home page (must be on VA network to view)](https://portalapps.vansoc.va.gov/EAS/Home).
+The VA SAVD WASA Coordination team can be contacted via email at `VASAVDWASACoordination[at]va.gov`. A request for WASA testing can be submitted on the [Security Assessment Portal home page (must be on VA network to view)](https://portalapps.vansoc.va.gov/EAS/Home).
```

Comment:
This URL has changed to https://saportal.va.gov/.

Reply:
This has been updated.